ros/src/twist_controller/cfg/DynReconf.cfg

```diff
@@ -8,8 +8,8 @@ PACKAGE = "twist_controller"
 
 gen = ParameterGenerator()
 
-gen.add("dyn_velo_proportional_control", double_t, 0, "Velocity proportional control constant", 0.1, 0, 4)
-gen.add("dyn_velo_integral_control", double_t, 0, "Velocity integral control constant", 0.01, 0, 4.0)
+gen.add("dyn_velo_proportional_control", double_t, 0, "Velocity proportional control constant", 0.3, 0, 4)
+gen.add("dyn_velo_integral_control", double_t, 0, "Velocity integral control constant", 0.1, 0, 4.0)
```

Comment:
This I gain is really large relative to P gain.  I was surprised to see it in the video, since I gains are typically at least one or two orders of magnitude smaller than P gain since it accumulates over time and can cause hunting from sensitivity to response delays.  We can try this gain for now, but need to keep an eye out for I term oscillations during testing.

Reply:
Yeah, I think I'm going to revert the I gain change. I can always add it later if it looks like it is really necessary.